dstep/translator/Type.d

```diff
@@ -350,9 +352,10 @@ body
         {
             if (children[dimension].kind == CXCursorKind.CXCursor_IntegerLiteral)
             {
+                auto token = tokenInsideSquareBrackets(cursor);
                 auto expansions = context.macroIndex.queryExpansion(children[dimension]);
 
-                if (expansions.length == 1)
+                if (expansions.length == 1 && token.kind == TokenKind.identifier)
```

Comment:
With this change the following code:

```c
#define foo 2
#define __FSID_T_TYPE struct { int __val[foo]; }
typedef  __FSID_T_TYPE __fsid_t;
typedef __fsid_t fsid_t;
```

Translates to:

```d
extern (C):

enum foo = 2;

struct __fsid_t
{
    int[__FSID_T_TYPE] __val;
}

alias fsid_t = __fsid_t;
```

Which has the same issue as this PR is trying to solve.